ts/Series/Variwide/VariwideSeries.ts

```diff
@@ -235,6 +239,18 @@ class VariwideSeries extends ColumnSeries {
             }
         }
     }
+
+    public getXExtremes(
+        xData: Array<number>|Types.TypedArray
+    ): RangeSelector.RangeObject {
+        const max = arrayMax(xData),
+            maxZ = this.getColumn('z')[xData.indexOf(max)];
+
+        return {
+            min: arrayMin(xData),
+            max: max + (!this.xAxis.categories ? maxZ : 0)
```

Comment:
It's slightly more readable and compact if we avoid the negation
```suggestion
            max: max + (this.xAxis.categories ? 0 : maxZ)
```